src/main/java/duke/Event.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+public class Event extends Task{
+    protected String at;
+
+    public Event(String description, char taskType, String at){
+        super(description);
+        this.taskType = taskType;
+        this.at = at;
+    }
+
+    public String getAt() {return at;}
```

Comment:
This method name falls on the short side. Consider a more [descriptive method name](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#not-too-long-not-too-short).